src/Raven.Server/Web/Studio/StudioTasksHandler.cs

```diff
@@ -50,16 +48,31 @@ public Task FullDataDirectory()
             {
                 if (PathUtil.IsSubDirectory(result, ServerStore.Configuration.Core.DataDirectory.FullPath) == false)
                 {
-                    result = $"The administrator has restricted databases to be created only under the {RavenConfiguration.GetKey(x => x.Core.DataDirectory)} directory: '{ServerStore.Configuration.Core.DataDirectory.FullPath}'.";
+                    result =
+                        $"The administrator has restricted databases to be created only under the {RavenConfiguration.GetKey(x => x.Core.DataDirectory)} directory: '{ServerStore.Configuration.Core.DataDirectory.FullPath}'.";
                 }
             }
+
+            var getNodesInfo = GetBoolValueQueryString("getNodesInfo", required: false) ?? false;
+            var info = new DataDirectoryInfo(ServerStore, result, getNodesInfo, ResponseBodyStream());
+            var urlPath = $"admin/studio-tasks/full-data-directory?path={path}&name={name}";
```

Comment:
Name and Path should be parameters to command. Endpoint url should be hardcoded inside the command.

Reply:
done